build/root/WORKSPACE

```diff
@@ -83,7 +83,7 @@ docker_pull(
     digest = "sha256:efc1d8a37f141869b7e45fa4e153ebea11502ee3d0ac29b25cd94cb02a40a91c",
```

Comment:
this should be updated to `sha256:a3b936c0fb98a934eecd2cfb91f73658d402b29116084e778ce9ddb68e55383e`

Reply:
Done. Thanks!